js/switcher.js

```diff
@@ -80,6 +80,25 @@ const SwitcherSingleResultView = (r, isSelected, selectResult) => {
 
 class Switcher extends Component {
   handleKeyDown(e) {
+    if (e.target === this.inputNode) {
+      if (e.which === 38) {
+        //arrow up
+        e.preventDefault(); // prevents caret from moving in input field
+        this.setState({
+          selectedIndex: Math.max(this.state.selectedIndex - 1, 0)
+        });
+      } else if (e.which === 40) {
+        // arrow down
+        e.preventDefault();
+        this.setState({
+          selectedIndex: Math.min(
+            this.state.selectedIndex + 1,
+            this.state.results.length - 1
+          )
+        });
+      }
+    }
+
```

Comment:
moved arrow up/down handling here so it's a little better experience – user can hold arrow on scroll through all list, instead of pushing it down every time